model/ResultsViewerService.php

```diff
@@ -31,7 +31,7 @@ class ResultsViewerService extends ConfigurableService
 {
     public const SERVICE_ID = 'taoOutcomeUi/resultsViewer';
 
-    public const OPTION_DEFAULT_ITEM_TYPE = 'defaultItemType';
+    public const OPTION_DEFAULT_ITEM_TYPE = 'qtiItem';
```

Comment:
**question**: could you clarify why you updating the option name here? Here is how it was looking in a compiled state before your change:
```
return new oat\taoOutcomeUi\model\ResultsViewerService(array(
    'defaultItemType' => 'qtiItem'
));
```

and here how it will look after:
```
return new oat\taoOutcomeUi\model\ResultsViewerService(array(
    'qtiItem' => 'qtiItem'
));
```